E2a-TRAS/AL23/README.md

```diff
@@ -0,0 +1,132 @@
+| [IFC4x3_AbRV-E2a-GL01](./GL01) | (RFI) Global Positioning RFI dataset | none     |
+| [IFC4x3_AbRV-E1-AL22](./AL22)  | (RFI) Two alignments without cant    | none     |
+
+
+
+
+## Usages, Constraints & Logic
+Other than the logic embedded by the IFC Entities & Concept Templates required for this test, **and** the constraints captured in the *Usages, Constraints & Logic* section of of precondition tests, **no additional constraints are applied**
+
+
+
+
+## Expected Results
+
+For certification of capabilities the only source will be:
+
+- n. 1 IFC file containing the information as requested. The file shall be named using the following syntax: `MVDCode`_`ExchangeCode`_`TestCode`_`SoftwareVendor`.`ifc` (Example: `IFC4.3AbRV_E2a_AL23_AmazingSoft.ifc`)
+
+
+
+
+## Validation criteria
+:zap: For this test case to be considered passed **all capabilities listed in this section**, and **the ones of pre-required tests** shall be verified, with no exception. :zap:
+
+### General
+
+- All the concept templates must be correctly implemented as presented in the validation criteria
+- At least 1 instance of each entity listed in [Itemised Roots](#Itemised-Roots) is present in the file
+- All validation criteria of the pre-required tests shall apply here too
+
+
+### Railway alignment (with cant)
+
+> **Acceptance criteria**: For the **Railway alignment (with cant)** capability, the validation procedure must verify that **all** the following validation criteria are satisfied.
+>
+> If present, all criteria listed in [Usages, Constraints & Logic](#Usages,-Constraints-&-Logic), and in the same section of precondition tests, shall be verified too.
+
+<details open><summary>Railway alignment (with cant)</summary> 
+
+| **ID**  | **CRITERIA**                                                     | **VALUE**                                   | **COMMENT**                |
+|---------|------------------------------------------------------------------|---------------------------------------------|----------------------------|
+| ALIG_01 | Alignments contained in file                                     | 2                                           |                            |
+| ALIG_02 | Component for Alignment 1_Primary route                          | 1 horizontal, 1 vertical, 1 cant            |                            |
+| ALIG_03 | Component for Alignment 2_Diverted route                         | 1 horizontal, 1 vertical, 1 cant            |                            |
+| ALIG_04 | The horizontal (H) layout is made only by these type of segments | straight line, circular arc, clothoid       |                            |
+| ALIG_05 | The vertical (V) layout is made only by these type of segments   | straight line, circular arc                 |                            |
+| ALIG_06 | The cant (C) layout is made only by these type of segments       | constant straight line, linear transition   |                            |
+
```

Comment:
I guess all the alignment files need to be checked for geometry? that means start point, end point, point list, etc.?

Reply:
yes, at list for testing the import